src/index.js

```diff
@@ -0,0 +1,101 @@
+import './style.css';
+
+const section = document.getElementById('todos');
+const input = document.createElement('input');
+const button = document.createElement('button');
+const inpButton = document.createElement('div');
+inpButton.classList = 'add-input';
+button.innerText = 'Add';
+input.classList = 'input';
+input.id = 'input';
+input.placeholder = 'Add to your list...';
+
+inpButton.append(input, button);
+
+button.classList = 'add-btn';
+section.append(inpButton);
+
+class Todo {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  static getTodos() {
+    let toDos;
+    if (localStorage.getItem('todos') === null) {
+      toDos = [];
+    } else {
```

Comment:
- [ ] Please add default array item that contains different keys like `description`, `completed`, and `index`.